pallets/gear/src/schedule.rs

```diff
@@ -1587,58 +1582,3 @@ impl<T: Config> Schedule<T> {
         }
     }
 }
-
-#[cfg(test)]
-mod test {
```

Comment:
wheres the test?

Reply:
It was test against attack vector that is not applicable for us 

https://github.com/casper-network/casper-wasm/pull/1